flang/lib/Lower/CharacterExpr.cpp

```diff
@@ -187,8 +187,10 @@ Fortran::lower::CharacterExprHelper::toExtendedValue(mlir::Value character,
       return materializeValue(fir::getBase(character));
     }
   }
-  if (!resultLen)
+  if (!resultLen) {
     mlir::emitError(loc, "no dynamic length found for character");
+    llvm::report_fatal_error("fatal");
```

Comment:
Why two errors? This code uses emitError in a lot of places. Maybe a `return` is required here?

Reply:
cheap debug leftover, I wanted a stack dump here instead of later when the empty mlir::Value is used. If there is a way to dump the user `loc` in `llvm::report_fatal_error` I would favor it to avoid returning an empty value.